libp2p/connmanager.nim

```diff
@@ -48,10 +48,11 @@ type
   PeerEventKind* {.pure.} = enum
     Left
     Joined
+    Identified
 
   PeerEvent* = object
     case kind*: PeerEventKind
-    of PeerEventKind.Joined:
+    of PeerEventKind.Joined, PeerEventKind.Identified:
```

Comment:
why do we need this?

Reply:
It might be fine as this case is minor, but it's better to add functionality only when needed.